packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -209,27 +209,28 @@ const actionsShape = PropTypes.arrayOf(
   })
 )
 
-const IndividualAction = [
+const Action = [
   provideState({
     computed: {
-      disabled: ({ item }, { disabled, userData }) =>
-        isFunction(disabled) ? disabled(item, userData) : disabled,
-      handler: ({ item }, { handler, userData }) => () =>
-        handler(item, userData),
-      icon: ({ item }, { icon, userData }) =>
-        isFunction(icon) ? icon(item, userData) : icon,
-      item: (_, { item, grouped }) => (grouped ? [item] : item),
-      label: ({ item }, { label, userData }) =>
-        isFunction(label) ? label(item, userData) : label,
-      level: ({ item }, { level, userData }) =>
-        isFunction(level) ? level(item, userData) : level,
+      disabled: ({ items }, { disabled, userData }) =>
+        isFunction(disabled) ? disabled(items, userData) : disabled,
+      handler: ({ items }, { handler, userData }) => () =>
+        handler(items, userData),
+      icon: ({ items }, { icon, userData }) =>
+        isFunction(icon) ? icon(items, userData) : icon,
+      items: (_, { item, grouped, selectedItems }) =>
+        selectedItems === undefined ? (grouped ? [item] : item) : selectedItems,
```

Comment:
We don't need to have 2 different props (`item` and `selectedItems`) for `Action`.

We could even handle the "single item" vs "items array" problem when *using* `Action`. I'm not sure if that one is actually better.